app/models/accomplishment.rb

```diff
@@ -0,0 +1,4 @@
+class Accomplishment < ApplicationRecord
+	has_many :accomplishments_members
```

Comment:
This is a many-to-many relationship through the `accomplishments_members` table.